.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: check-toml
+  - repo: https://github.com/psf/black
```

Comment:
FYI, black now has a special pre-commit implementation so you can change the URL to https://github.com/psf/black-pre-commit-mirror (but need to uprev the black version first with `pre-commit autoupdate`).